src/app/components/HomePage/Info.md

```diff
@@ -0,0 +1,11 @@
+FYT Code Cup là một kỳ thi đặc biệt, được tài trợ bởi Trung tâm Tài năng trẻ FPT (FPT Young Talents).<br /><br />
+
+Kỳ thi sẽ được diễn ra trong hai ngày:<br />
```

Comment:
Don't use `<br />` here. This is a MD file, not a HTML file.

The proper way to add new line is to use two new line characters.

Reference: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet